swarm/src/connection/pool.rs

```diff
@@ -495,6 +497,10 @@ where
     ) {
         let connection = connection.extract();
 
+        let span = tracing::error_span!("peer id:", %obtained_peer_id, connection_id = tracing::field::Empty);
+        let _guard = span.enter();
+        span.record("connection id", tracing::field::display(id));
```

Comment:
You already have the connection id available when you construct the swarm, no need to use `tracing::field::Empty`.